index.js

```diff
@@ -61,7 +61,7 @@ export function locatePathSync(
 
 	for (const path_ of paths) {
 		try {
```

Comment:
Then you don't really need the try/catch here.

Reply:
I thought the `try/catch` _may_ still be useful. Just because it doesn't throw an exception if the file doesn't exist doesn't mean it'll never throw

Reply:
Good point 👍

Reply:
I actually just tried this experimentally and wanted to mention that ENOTDIR is still thrown, but ENOENT doesn't throw anymore since adding the `throwIfNoEntry`. Either way, a try/catch is still needed unfortunately